internals/scripts/clean.js

```diff
@@ -13,68 +13,36 @@ if (!test('-e', 'internals/templates')) {
 
 process.stdout.write('Cleanup started...');
 
-// Cleanup components folder
+// Reuse existing LanguageProvider and i18n tests
+mv('app/containers/LanguageProvider/tests', 'internals/templates/containers/LanguageProvider');
+cp('app/tests/i18n.test.js', 'internals/templates/tests/i18n.test.js');
+
+// Cleanup components/
 rm('-rf', 'app/components/*');
 
-// Cleanup containers folder
+// Handle containers/
 rm('-rf', 'app/containers/*');
-mkdir('-p', 'app/containers/App');
-mkdir('-p', 'app/containers/NotFoundPage');
-mkdir('-p', 'app/containers/HomePage');
-cp('internals/templates/appContainer.js', 'app/containers/App/index.js');
-cp('internals/templates/constants.js', 'app/containers/App/constants.js');
-cp('internals/templates/notFoundPage/notFoundPage.js', 'app/containers/NotFoundPage/index.js');
-cp('internals/templates/notFoundPage/messages.js', 'app/containers/NotFoundPage/messages.js');
-cp('internals/templates/homePage/homePage.js', 'app/containers/HomePage/index.js');
-cp('internals/templates/homePage/messages.js', 'app/containers/HomePage/messages.js');
-
-// Handle Translations
-rm('-rf', 'app/translations/*')
-mkdir('-p', 'app/translations');
-cp('internals/templates/translations/en.json',
-  'app/translations/en.json');
+mv('internals/templates/containers', 'app');
```

Comment:
Doesn't work on Windows, but it will work if you remove `containers` folder completely: `rm('-rf', 'app/containers');`. Same true for `translations`.